lib/userAgent.js

```diff
@@ -64,14 +64,31 @@ function getVersion (data) {
 
 function getDeviceType (data) {
   if (getFamily(data.device) || isMobileOS(data.os)) {
-    return 'mobile'
+    if (isTablet(data)) {
+      return 'tablet'
+    } else {
+      return 'mobile'
+    }
   }
 }
 
 function isMobileOS (os) {
   return os.family in MOBILE_OS_FAMILIES
 }
 
+function isTablet(data) {
+  // 'tablets' are iPads and Android devices with no word 'Mobile' in them.
```

Comment:
I didn't believe this until I found a supporting article. Might be helpful to reference? 

https://webmasters.googleblog.com/2011/03/mo-better-to-also-detect-mobile-user.html.